aws/resource_aws_vpn_connection_test.go

```diff
@@ -325,6 +359,38 @@ func testAccAwsVpnConnectionConfigUpdate(rInt, rBgpAsn int) string {
 	`, rBgpAsn, rInt)
 }
 
+func testAccAwsVpnConnectionConfigTunnelOptions(rBgpAsn int) string {
+	return fmt.Sprintf(`
+		resource "aws_vpn_gateway" "vpn_gateway" {
```

Comment:
Nitpick: I don't think we have a style guide I can point to, but can you please ensure the Terraform configuration inside the testing here is fully to the left with 2 space indentation? We're trying to standardize going forward. Thanks!